src/features/shared-link-settings-modal/AllowDownloadSection.js

```diff
@@ -35,23 +38,38 @@ const AllowDownloadSection = ({
         </div>
     );
 
+    const label = classification ? `"${classification}"` : null;
```

Comment:
Is this variable needed? Can below be label: classification || null. Also if null, then the display message on tooltip will say null. In other places we just say that "disabled due to an applied access policy" since having a classification doesn't do the disabling, but its the access policy. So maybe for consistency we should match what we say in other tooltips.